### DISCUSSION TITLE
Can i join team sir?

### DISCUSSION THREAD

**Discussion by xyz-sundram** [ROM-builders/temporary#21707] [General]
Github id : xyz-sundram
Telegram id : @Nerd_Sundram

Device tree: https://github.com/xyz-sundram/device_xiaomi_twolip

Successful build :
https://cirrus-ci.com/build/6296699124056064

Thankyou sir.

**Comment by Apon77** [comment]
Please check #learn instruction from telegram group.

**Reply by xyz-sundram** [reply]
So sir am i lacking in respecting commit history? I already knew cherry pick and Git CLI

**Reply by xyz-sundram** [reply]
Can i join now sir?

I have updated my device tree names, so here are the updated links

Device tree: https://github.com/xyz-sundram/device_xiaomi_tulip
Vendor tree: https://github.com/xyz-sundram/vendor_xiaomi_tulip
Kernel tree: https://github.com/xyz-sundram/android_kernel_xiaomi_sdm660

My successful builds:
https://cirrus-ci.com/build/6296699124056064
https://cirrus-ci.com/build/4805510436749312
